bin/start.sh

```diff
@@ -105,5 +103,5 @@ else
 fi
 
 echo "Triggering Spark Submit job"
-echo_formatted "${command}"
-$command
+echo ${command} "$@"
+${command} "$@"
```

Comment:
These changes were made in shell script to allow it pass properties with whitespaces to the gcloud command.